app/scripts/views/compose-message.js

```diff
@@ -94,6 +95,13 @@ define([
         return d.textContent;
       });
       return lines.join('\n');
+    },
+    
+    _sendByEnter: function (e) {
+      if(e.key === 'Enter' && localStorage.getItem('sendByEnter')) {
```

Comment:
I think moving the `localStorage` bits to the `initialize` function would save us some milliseconds (in low-end terminals).

If we initialize the data, we can access directly a variable, instead of getting the data from the sync storage engine.

What do you think @Loirooriol?